### DISCUSSION TITLE
Why does the README show `<~>` as a Java ligature?

### DISCUSSION THREAD

**Discussion by schuelermine** [tonsky/FiraCode#1582] [General]
As far as I’m aware that’s not Java syntax.

**Comment by wopian** [comment]
It's an IntelliJ IDEA code-folding ligature for Java generics. IntelliJ uses it to indicate that the generic is the same as the left hand side type declaration. Clicking the tilde reveals the type actually written in the source code.

**Reply by schuelermine** [reply]
Thanks
